server/controllers/customers/csv.customers.js

```diff
@@ -0,0 +1,68 @@
+const model = require('../../models')
```

Comment:
This is just a **nitpick**:
The file naming convention is related more with jQuery. I recommended following Google's style guide for naming files and be consistent with it https://google.github.io/styleguide/jsguide.html#file-name. 
for example it could be `csv_customers.js` or `csv-customers.js`